chia/consensus/blockchain.py

```diff
@@ -988,6 +988,11 @@ async def get_block_records_at(self, heights: List[uint32], batch_size: int = 90
             records.extend(res)
         return records
 
+    def try_block_record(self, header_hash: bytes32) -> Optional[BlockRecord]:
```

Comment:
This name isn't clear for me.  It reads like it is trying the block record...  to see if it is...  valid?  Perhaps `maybe_get_block_record()` or `optional_get_block_record()`?  I dunno.

Reply:
this name is already part of the current `BlockchainInterface` and I'm hesitant to change these names at the same time as I move them to a new protocol.

Reply:
Ah, sorry.  I didn't look enough.